modular_skyrat/modules/company_imports/code/armament_datums/nri_military_surplus.dm

```diff
@@ -0,0 +1,136 @@
+/datum/armament_entry/company_import/nri_surplus
+	category = NRI_SURPLUS_COMPANY_NAME
+	company_bitflag = CARGO_COMPANY_NRI_SURPLUS
+
+// Various NRI clothing items
+
+/datum/armament_entry/company_import/nri_surplus/clothing
+	subcategory = "Clothing Supplies"
+
+/datum/armament_entry/company_import/nri_surplus/clothing/uniform
+	item_type = /obj/item/clothing/under/costume/nri
+	lower_cost = CARGO_CRATE_VALUE * 1.3
+	upper_cost = CARGO_CRATE_VALUE * 1.7
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nri_surplus/clothing/boots
+	item_type = /obj/item/clothing/shoes/russian
+	lower_cost = CARGO_CRATE_VALUE * 0.3
+	upper_cost = CARGO_CRATE_VALUE * 1
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nri_surplus/clothing/boots
+	item_type = /obj/item/clothing/shoes/russian
+	lower_cost = CARGO_CRATE_VALUE * 0.3
+	upper_cost = CARGO_CRATE_VALUE
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nri_surplus/clothing/cap
+	item_type = /obj/item/clothing/head/soft/nri_larp
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nri_surplus/clothing/ushanka
+	item_type = /obj/item/clothing/head/costume/ushanka
+	lower_cost = CARGO_CRATE_VALUE * 0.5
+	upper_cost = CARGO_CRATE_VALUE
+	stock_mult = 2
+	interest_addition = COMPANY_INTEREST_GAIN_PITIFUL
+
+/datum/armament_entry/company_import/nri_surplus/clothing/belt
+	item_type = /obj/item/storage/belt/military/nri
+	lower_cost = CARGO_CRATE_VALUE
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nri_surplus/clothing/backpack
+	item_type = /obj/item/storage/backpack/nri
+	lower_cost = CARGO_CRATE_VALUE
+	upper_cost = CARGO_CRATE_VALUE * 1.5
+	interest_required = PASSED_INTEREST
+
+/datum/armament_entry/company_import/nri_surplus/clothing/helmet
+	item_type = /obj/item/clothing/head/helmet/rus_helmet
```

Comment:
I'd ask you to properly finish that one salt PR by making this helmet into the /nri subtype and making Cargo only sell the /tg/ armor.